sqlmesh/dbt/basemodel.py

```diff
@@ -296,22 +304,52 @@ def __init__(self, context: DbtContext, config: BMC):
     @property
     def enriched_config(self) -> BMC:
         if self._rendered_sql is None:
+            self._enriched_config = self._update_with_sql_config(self._enriched_config)
             self.render()
             self._enriched_config._dependencies = self._enriched_config._dependencies.union(
                 self._captured_dependencies
             )
         return self._enriched_config
 
+    @property
+    def parsed_sql(self) -> nodes.Template:
```

Comment:
The name is confusing. I expected it to return SQLGlot AST. `parsed_jinja` maybe?

Reply:
sure